src/redux/book/books.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-alert */
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+// Actions
```

Comment:
- [ ] You have disabled too many linters, Linters help us to write clean code, please try to fix those errors  instead of disabling them, I am giving this as required changes because you disabled many, kindly reduce the number of linters rule you disable